WordPress/Sniffs/Theme/DeregisterScriptSniff.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Sniff to check theme for wp_deregister_style()
+ *
+ * @category Theme
+ * @package  PHP_CodeSniffer
+ * @author   Simon Prosser <pross@pross.org.uk>
+ */
+class WordPress_Sniffs_Theme_DeregisterScriptSniff extends WordPress_Sniffs_Functions_FunctionRestrictionsSniff
+{
+
+	/**
+	 * Exclude groups
```

Comment:
I think @jrfnl means `public $exclude` because it is not needed as it is defined in `WordPress_Sniffs_Functions_FunctionRestrictionsSniff` and we are not changing the value.